packages/teleport-uidl-validator/src/uidl-schemas/component.json

```diff
@@ -299,6 +299,12 @@
         "key": {"type": "string"},
         "dependency": {"$ref": "#/definitions/componentDependency"},
         "style": {"$ref": "#/definitions/styleDefinitions"},
+        "abilities": {
+          "type": "object",
+          "properties": {
+            "link": {"$ref": "#/definitions/linkDefinition"}
+          }
+        },
```

Comment:
TODO: Update project-schema at `https://docs.teleporthq.io/uidl-schema/v1/project.json` to support the new custom-code and abilities. UIDL's downloaded from playground still point to it. Nice to have both in sync :)